cli/tests/pcluster/cli/test_entrypoint/TestParallelClusterCli/test_helper/pcluster-help.txt

```diff
@@ -1,50 +1,75 @@
 usage: pcluster [-h]
```

Comment:
It's generally easy enough to do -- do you want to ensure that it exists? (this could be done from the model perspective) or do you want to lock-in what the text is?

Reply:
lock-in what the text is

Reply:
Added a SIM for this as as sub-task of reviewing the API